application/commandline_interface.py

```diff
@@ -115,8 +119,8 @@ def display_main_menu(self): # TESTED AND WORKING
                 username, password = self.request_login_details()  # Request user login details
                 # Handle Login
                 if self.login_service.login(username, password):
-                    self.display_user_menu(username)
-                    #self.display_test_menu(username)
+                    #self.display_user_menu(username)
+                    self.display_test_menu(username)

```

Comment:
This probably needs to be the other way around for the main branch

Reply:
My bad, I'll change that!